index.js

```diff
@@ -28,6 +28,16 @@ module.exports = {
         "no-console": "error",
         "no-only-tests/no-only-tests": "error",
 
+        ////////// No Enums ////////////////
+
+        "no-restricted-syntax": [
```

Comment:
this should please go to the file `typescript.js`, which we extend for projects that use typescript. It also only provides overrides for `.ts` and `.tsx`, where we should put this.

Reply:
moved in [46684c](https://github.com/pmedianetwork/eslint-config/pull/43/commits/46684cf8ff9e3bdce8c4d830afb5072f06b38cc8)